.github/workflows/test_periodic.yml

```diff
@@ -6,8 +6,8 @@ on:
   workflow_dispatch:
 
 env:
-  GO_VERSION: 1.15.1
-  K8S_VERSION: v1.19.0
```

Comment:
does this mean we wont be running against v1.19.0 any more? mostly just curious so I can learn more what is going on.

Reply:
seems like a lot more going on with this PR than just updating a job to run on master vs v1.19.0. Is it worth adding any details in the commit message? As a new person to the project, I'm not totally following all the changes happening just by looking at the diff.

Reply:
this is a periodic job that runs every day, I use it to gather feedback and check the status.
It was used until now to test dual stack, but once it was stable enough we moved the dualstack to job to presubmit so it can gate in the code.
I want to check the latest changes in dual stack, so we need to get the master branch for that, but this is unrelated to the presubmit jobs, we can just add more jobs here if we want

Reply:
> seems like a lot more going on with this PR than just updating a job to run on master vs v1.19.0. Is it worth adding any details in the commit message? As a new person to the project, I'm not totally following all the changes happening just by looking at the dif

yeah, well , just was trying and error, should add a WIP to the job, will update once this is ready

Reply:
cool, thanks for the explanations. Let me know if you have any way I can help with these types of jobs.

Reply:
if you are interested in these jobs, once I have it working we can do a videocall and go through all the details, 
I feel it will be easier and more efficient